README.md

```diff
@@ -1 +1,113 @@
 # hermione-url-decorator
+
+[![Build Status](https://travis-ci.org/gemini-testing/hermione-url-decorator.svg?branch=master)](https://travis-ci.org/gemini-testing/hermione-url-decorator)
+[![Coverage Status](https://img.shields.io/coveralls/gemini-testing/hermione-url-decorator.svg?style=flat)](https://coveralls.io/r/gemini-testing/hermione-url-decorator?branch=master)
+
+Plugin for [hermione](https://github.com/gemini-testing/hermione) which is intended to add additional parameters to the test browser urls. You can read more about plugins at [documentation](https://github.com/gemini-testing/hermione#plugins).
+
+## Installation
+
+```bash
+npm install hermione-url-decorator
+```
+
+## Configuration
+
+* **enabled** (optional) `Boolean` – enable/disable the plugin; by default plugin is enabled
+* **url** (optional) `Object` - the list of url parameters, which will be added in each test url
+    * **query** (optional) `Object` - the list of query parameters
+        * **queryParams** (optional) `Object` - names of query parameters
+            * **value** (optional) `String` - value of query parameter
+            * **concat** (optional) `String|Boolean` - enable/disable concatenation of query parameters or specify the delimeter; by default is `false`
```

Comment:
enable/disable concatenation of query parameters; if the value is string, existing query value will be concatenated with the defined value by the specified string

?